core-plugins/widgets/File-batchsource.json

```diff
@@ -28,6 +28,16 @@
           "name": "format",
           "widget-attributes": {
             "plugin-type": "validatingInputFormat"
+          },
+          "plugin-function": {
```

Comment:
I think it should not be tied to any widget, try using format like below ?
https://github.com/data-integrations/database-plugins/blob/fabc900f9dde17b11c7d5a884f9fc52e5289f037/mysql-plugin/widgets/Mysql-batchsource.json#L106